integration_test/tests/function_tests.rs

```diff
@@ -0,0 +1,309 @@
+mod common;
```

Comment:
I added this to show SQLPrepareW and SQLExecute in action. It can probably be cleaned up and some things shared with the power bi tests (I just copied the file and deleted a bunch of stuff) not needed for this.